tests/integ/sagemaker/jumpstart/private_hub/model/test_jumpstart_private_hub_model.py

```diff
@@ -122,7 +122,6 @@ def test_jumpstart_hub_gated_model(setup, add_model_references):
     assert response is not None
 
 
-@pytest.mark.skip(reason="blocking PR checks and release pipeline.")
 def test_jumpstart_gated_model_inference_component_enabled(setup, add_model_references):
```

Comment:
can you add unit tests too?

Reply:
This integ test correctly caught the issue, we just incorrectly skipped it instead of root causing

Reply:
yes but we shouldn't need integ test to catch errors like this imo. the unit tests should be sufficient.

integ tests are for testing integrations with other services. This issue you're fixing is really just isolated to the PySDK.